src/main/resources/templates/index.html

```diff
@@ -8,7 +8,7 @@
   <!-- Bootstrap CSS -->
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" type="text/css" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
   <link href="styles.css" rel="stylesheet">
-  <title>Hello, world!</title>
+  <title>WeatherRunner</title>

```

Comment:
There isn't much left to review, so I've updated their page title to reflect the project name